### ISSUE TITLE
[iOS] Provide accessibility information for a page source

### ISSUE THREAD

**Issue by TheDarkestDay** [appium/appium#15232]
## The problem

Include `accessible` field into page source representation 

## Details

It seems as if with help of Appium we can even write some automative checks for accessibility information. For instance, we can check accessibility label presence on certain elements. 

However, technically not every UI element should be a subject for such checks. Some of them are excluded from the accessibility layer completely by having their `is_accessibility_element` set to `false`. This information can be used to actually distinguish elements which require checking. 

WebDriverAgent used by XCUITest Appium driver has this `accessible` value in its source code mentioned:
https://github.com/appium/WebDriverAgent/blob/a1824895274c60798b506f31054e4ce1e0400c71/WebDriverAgentLib/Categories/XCUIApplication%2BFBHelpers.m#L151
Moreover, there is an API which allows you to grab `accessible` attribute of some single UI element.

With this being said, it would be great to provide this `accessible` field right into page source representation in order to be able to get this data for all of the nodes simultaneously. 

Any thoughts on this? Do you guys accept any contributions in this regard?

**Comment by mykola-mokhnach** [comment]
I think this is doable. Feel free to add a PR

**Comment by mykola-mokhnach** [comment]
The requested functionality has been added to appium@beta. Thanks for creating the PR
